Source/EventStoreResourceTypeRepresentation.cs

```diff
@@ -0,0 +1,42 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Dolittle. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ * --------------------------------------------------------------------------------------------*/
+ 
+using System;
+using System.Collections.Generic;
+using Dolittle.Resources;
+
+namespace Dolittle.Runtime.Events.MongoDB
+{
+    /// <inheritdoc/>
+    public class EventStoreResourceTypeRepresentation : IRepresentAResourceType
+    {
+        IDictionary<Type, Type> _bindings;
+        
+        /// <inheritdoc/>
+        public ResourceType Type => "eventStore";
+        /// <inheritdoc/>
+        public ResourceTypeImplementation ImplementationName => "MongoDB";
+        /// <inheritdoc/>
+        public Type ConfigurationObjectType => typeof(EventStoreConfiguration);
+        /// <inheritdoc/>
+        public IDictionary<Type, Type> Bindings {
```

Comment:
This could be simplified - as it is static: 

```suggestion
public IDictionary<Type, Type> Bindings => new Dictionary<Type,Type> {
   {typeof(Dolittle.Runtime.Events.Store.IEventStore), typeof(Dolittle.Runtime.Events.Store.MongoDB.EventStore)},
   {typeof(Dolittle.Runtime.Events.Store.Relativity.IGeodesics), typeof(Dolittle.Runtime.Events.Relativity.MongoDB.Geodesics)},
   {typeof(Dolittle.Runtime.Events.Processing.IEventProcessorOffsetRepository), typeof(Dolittle.Runtime.Events.Processing.MongoDB.EventProcessorOffsetRepository)},
   
}
```

Then you don't need to do the InitializeBindings() - nor a Null check.